src/libs/Navigation/Navigation.js

```diff
@@ -95,7 +96,7 @@ function goBack(shouldOpenDrawer = true) {
  */
 function navigate(route = ROUTES.HOME) {
     if (!navigationRef.isReady()) {
-        console.debug('[Navigation] navigate failed because navigation ref was not yet ready');
+        Log.info('[Navigation] navigate failed because navigation ref was not yet ready');
```

Comment:
`hmmm`? Should we also include the `route` so it is easier to see what we are trying to navigate to when not ready?